vm/src/vm/runners/cairo_runner.rs

```diff
@@ -528,6 +528,144 @@ impl CairoRunner {
+
+        // self.program
+        //     .shared_program_data
+        //     .hints_collection
+        //     .iter()
+        //     .flat_map(|(pc, hints)| {
+        //         self.program
+        //             .shared_program_data
+        //             .hints_collection
+        //             .get_hint_range_for_pc(pc)
+        //             .and_then(|hint_range| hint_range)
+        //             .map(|(start, len)| {
+        //                 let len = len.get();
+        //                 let end = start + len;
+        //                 if end <= hints.len() {
+        //                     Some((pc, &hints[start..end]))
+        //                 } else {
+        //                     None
+        //                 }
+        //             })
+        //     })
+        //     .flatten()
+        //     .map(|(_pc, hint)| {
+        //         hint_executor
+        //             .compile_hint(
+        //                 &hint[0].code,
+        //                 &hint[0].flow_tracking_data.ap_tracking,
+        //                 &hint[0].flow_tracking_data.reference_ids,
+        //                 references,
+        //             )
+        //             .map_err(|_| VirtualMachineError::CompileHintFail(hint[0].code.clone().into()))
+        //     })
+        //     .collect()
+
+        // self.program
+        //     .shared_program_data
+        //     .hints_collection
+        //     .iter()
+        //     .flat_map(|(_pc, hints)| {
+        //         hints.iter().map(|hint| {
+        //             hint_executor
+        //                 .compile_hint(
+        //                     &hint.code,
+        //                     &hint.flow_tracking_data.ap_tracking,
+        //                     &hint.flow_tracking_data.reference_ids,
+        //                     references,
+        //                 )
+        //                 .map_err(|_| VirtualMachineError::CompileHintFail(hint.code.clone().into()))
+        //         })
+        //     })
+        //     .collect::<Result<Vec<_>, _>>()
+        // let mut hint_data = Vec::new();
+
+        // self.program
+        //     .shared_program_data
+        //     .hints_collection
+        //     .iter()
+        //     .try_for_each(|(_pc, hints)| {
+        //         hints
+        //             .iter()
+        //             .try_for_each(|hint| -> Result<(), VirtualMachineError> {
+        //                 let compiled_hint = hint_executor.compile_hint(
+        //                     &hint.code,
+        //                     &hint.flow_tracking_data.ap_tracking,
+        //                     &hint.flow_tracking_data.reference_ids,
+        //                     references,
+        //                 )?;
+
+        //                 hint_data.push(Box::new(compiled_hint) as Box<dyn Any>);
+
+        //                 Ok(())
+        //             })
+        //     })?;
+
+        // Ok(hint_data)
```

Comment:
I gather this came from iterating solutions and slipped. Could you remove it before merge?

Reply:
Oops. Good catch. Removed!